content/pricing/billing.textile

```diff
@@ -33,3 +33,17 @@ To update your billing details:
 You can update your payment details and view your invoices in this screen.
 
 It's also possible to have your invoices sent to an accounts department by filling in the *Optional billing email address*.
+
+h2(#alerts). Billing alerts
+
+Billing alerts notify you by email when your monthly spending exceeds a set amount on the accounts where you have paid "packages":/docs/pricing#packages (standard or pro). This feature helps you monitor costs and avoid unexpected charges.
```

Comment:
```suggestion
Billing alerts notify you by email when your monthly spending exceeds a set amount if you're on a Standard or Pro package. This feature helps you monitor costs and avoid unexpected charges.
```

There's a few too many references to multiple accounts here, which is rare for most users.